MinimalCirclesPath.py

```diff
@@ -94,13 +99,49 @@ def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
     return coordList
        
 
-
+def plot_path_on_us_map(coordList):
+    # Load the US map from the GeoJSON file
+    us_map = gpd.read_file('us.geojson')
```

Comment:
isn't this already read once and stored in the `gdf` variable?

Reply:
> isn't this already read once and stored in the `gdf` variable?

correct, will fix